### PR TITLE
docs update: esdt supply, api elrond com, remove deprecated comparisons

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
#### Description of the pull request (what is new / what has changed)
- documented the endpoint for fetching a token's supply, burnet and minted values
- removed deprecated comparisons
- documented `api.elrond.com`

#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [x] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/rest-api/api-elrond-com.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+id: api-elrond-com
+title: api.elrond.com
+---
+
+## About api.elrond.com
+
+`api.elrond.com` is a wrapper over `gateway.elrond.com` that brings a robust caching mechanism, alongside Elasticsearch 
+historical queries support, tokens media support, delegation&staking data, and many others.
```
Comment (andreibancioiu) [inline docs/sdk-and-tools/rest-api/api-elrond-com.md:9]: `delegation & staking`

File: docs/sdk-and-tools/rest-api/gateway-overview.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+}
+```
+
+:::important
+When describing each HTTP endpoint on the following pages, the basic structure of the response is **simplified for brevity,** and, in general, only the actual payload of the response is depicted.
+:::
+
+## **REST Client playground**
+
+If you wish to rapidly test the API endpoints we have created some [playground files](https://github.com/ElrondNetwork/elrond-sdk/tree/master/proxy-playground) for [Visual Studio Code REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client). These files contain examples for using the API.
```
Comment (andreibancioiu) [inline docs/sdk-and-tools/rest-api/gateway-overview.md:73]: They are a bit deprecated (I think, since they are in the archived repository).

**Review by andreibancioiu — APPROVED** [review]

File: docs/sdk-and-tools/rest-api/api-elrond-com.md (hunk 1 above)
Comment (bogdan-rosianu) [reply docs/sdk-and-tools/rest-api/api-elrond-com.md]: removed

File: docs/sdk-and-tools/rest-api/gateway-overview.md (hunk 2 above)
Comment (bogdan-rosianu) [reply docs/sdk-and-tools/rest-api/gateway-overview.md]: removed

**Review by andreibancioiu — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

File: docs/sdk-and-tools/rest-api/rest-api.md [hunk 3]
```diff
@@ -1,71 +1,17 @@
...
 id: rest-api
-title: REST API
+title: REST API overview
 ---
 
-Overview of Elrond REST API
+## Introduction
 
-The Elrond REST API can be used by any application - dApp, desktop or server application - to interact with the Blockchain.
+Elrond has 2 layers of Rest APIs that can be publicly accessed. Both of them can be recreated by anyone that
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/sdk-and-tools/rest-api/rest-api.md]: changed for all pages

**Review by raduchis — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**